src/main/routes/enter-claim-number.ts

```diff
@@ -0,0 +1,16 @@
+import * as express from 'express'
+import { ErrorHandling } from 'common/errorHandling'
+import { Paths } from 'app/paths'
+/* tslint:disable:no-default-export */
+
+export default express.Router()
+  .get(Paths.enterClaimNumberPage.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response): Promise<void> => {
+      res.render('enter-claim-number')
+    })
+  )
+  .post(Paths.enterClaimNumberPage.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response): Promise<void> => {
+      res.render('enter-claim-number')
```

Comment:
That does not look right, we should take them either to CMC login page or MCOL page depending what reference user put in.